libs/react-helpers/src/lib/grid/date-range-filter.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen, waitFor, act } from '@testing-library/react';
+import type { DateRangeFilterProps } from './date-range-filter';
+import { DateRangeFilter } from './date-range-filter';
+
+const commonProps = {
+  filterChangedCallback: jest.fn(),
+};
+
+describe('DateRangeFilter', () => {
+  it('should be properly rendered', async () => {
+    const defaultRangeFilter = {
+      start: '2023-02-14T13:53:01+01:00',
+      end: '2023-02-21T13:53:01+01:00',
+    };
+    const displayStartValue = '2023-02-14T12:53:01.000';
+    const displayEndValue = '2023-02-21T12:53:01.000';
+    render(
```

Comment:
@macqbat I've pushed some changes to this test to simplify the code. Please note:

- removed `act`. testing-library/react automatically wraps with act when you use `render`. Normally you shouldn't need to wrap components with act when rendering. We often do when working with AgGrid as there are multiple events that update state after AgGrid Mounts
- Removed storing the container element in a variable. Just use  `screen` where possible
- Combined input assertions. getByLabelText will return the `<input>` for a corresponding label. It will error if the input isn't found so its safe to immediately check the value on it too.

Reply:
nice, thanks